### ISSUE TITLE
About RankMix implementation in paper.

### ISSUE THREAD

**Issue by ing907** [liupei101/PseMix#5]
Hello, thank you for your great work and for sharing the source code. All of your code, including RankMix and ReMix, has been very helpful.

However, I have a question regarding the RankMix implementation. I noticed that the self-training proposed in the RankMix paper is not included in this repository. I am curious whether the experiments for RankMix were conducted without self-training, as it is in this repository, or if self-training was included in those experiments in your paper.

Thank you.

**Comment by liupei101** [comment]
Hi, ing907. Thanks for your interest and attention. 

As you know, the self-training part of RankMix is not implemented in our repo. We did not run the experiments for a self-training version of RankMix. It is largely because 
1. We failed to access the source code of RankMix during our study period (finished in July 2023). So we choose to reproduce the core idea of RankMix. In the absence of the official source code, an in-house implementation for the self-training part could cause potential bias. 
2. We focus more on the augmentation technique, so the methods that can be decoupled from the training process are mainly compared in our experiments.
